src/main/java/levy/gameoflife/GameOfLife.java

```diff
@@ -0,0 +1,75 @@
+package levy.gameoflife;
+
+public class GameOfLife {
+    private int width;
+    private int height;
+    int[][] grid;
```

Comment:
private

Reply:
when i make private then the grid in my test doesnt have acess - the Ide gave me the option of packaged-private what is that? does that help?

Reply:
Make a method that allows you to pass it. Don't access set the value directly like that.

Reply:
just a setter no getter right?

Reply:
Unless you think you need both...

Reply:
i mean getter never used so no